js-exercises/fibonacci-iter/fibonacciIter.test.js

```diff
@@ -0,0 +1,26 @@
+import { fibonacciIter } from './fibonacciIter';
+
+describe('fibonacciIter', () => {
+  test('should be an iterable', () => {
+    const iterator = fibonacciIter[Symbol.iterator]();
+    expect(typeof fibonacciIter[Symbol.iterator]).toBe('function');
+    expect(typeof iterator.next).toBe('function');
+    expect(iterator.next()).toHaveProperty('value');
+    expect(iterator.next()).toHaveProperty('done');
+  });
+
+  test('should return fibonacciIter series', () => {
+    const iterator = fibonacciIter[Symbol.iterator]();
+    expect(iterator.next().value).toBe(1);
+    expect(iterator.next().value).toBe(2);
```

Comment:
Fibonacci series is 1, 1, 2, ...
And not 1, 2 ...

Please update test case, and logic accordingly :)